packages/core/src/shared/logger/logger.ts

```diff
@@ -5,7 +5,16 @@
 
 import * as vscode from 'vscode'
 
-export type LogTopic = 'crashMonitoring' | 'dev/beta' | 'notifications' | 'test' | 'childProcess' | 'lsp' | 'unknown'
+export type LogTopic =
+    | 'crashMonitoring'
+    | 'dev/beta'
+    | 'notifications'
+    | 'test'
+    | 'childProcess'
+    | 'lsp'
+    | 'QWorkspaceLsp'
```

Comment:
should these be `amazonqWorkspaceLSP` or is that too long? I feel like we need to have conventions on whether we use q vs amazonq since both are used in the codebase 🤔

Reply:
That feels more consistent. Better to start verbose and look for opportunities to shorten later.